src/npctalk.cpp

```diff
@@ -1481,10 +1481,11 @@ talk_topic dialogue::opt( dialogue_window &d_win, const std::string &npc_name,
         response_hotkeys.clear();
         input_event evt = ctxt.first_unassigned_hotkey( queue );
         for( talk_response &response : responses ) {
-            response_lines.emplace_back( response.create_option_line( *this, evt ) );
+            const talk_data td = response.create_option_line( *this, evt );
```

Comment:
```suggestion
            const talk_data &td = response.create_option_line( *this, evt );
```

A reference here should be fine, right?